.github/workflows/sca.yaml

```diff
@@ -0,0 +1,114 @@
+# yamllint disable rule:line-length
+
+name: Static Code Analysis
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        operating-system:
+          - ubuntu-latest
+        php-version:
+          - 7.4
+
+    name: Static Code Analysis
+
+    runs-on: ${{ matrix.operating-system }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-${{ matrix.composer-flags }}
+          restore-keys: |
+            composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-
+            composer-${{ runner.os }}-${{ matrix.php-version }}-
+            composer-${{ runner.os }}-
+            composer-
+
+      - name: Install dependencies
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
+          retry_wait_seconds: 30
+          command: |
+            composer update --optimize-autoloader --no-interaction --no-progress ${{ matrix.composer-flags }}
+
+      - name: Report versions
+        run: |
+          composer info -D
+
+      - name: Install dev-tools
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
+          retry_wait_seconds: 30
+          command: |
+            ./dev-tools/install.sh
+
+      - name: Run checks
```

Comment:
> It seems we're good to go: https://github.com/sanmai/PHP-CS-Fixer/runs/1447402026?check_suite_focus=true

![Screenshot from 2020-11-24 18-17-05](https://user-images.githubusercontent.com/2716794/100129087-5cf2b180-2e81-11eb-9c8b-bfccbc82d0c8.png)
what is line 8?

Reply:
Just the noise from GH actions wrapper. You can see it at every step.

I think they're trying to tell us which shell we're using, and with which arguments.